ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -125,6 +125,27 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-assisted-operator
+- as: e2e-metal-assisted-operator-bmh
+  steps:
+    cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        BMC_DRIVER=redfish-virtualmedia
+        NUM_WORKERS=2
```

Comment:
Wondering if it would be enough, at least for now, to run with just masters. It should make tests go faster and results should be the same.